packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -100,13 +101,17 @@ export class ModelInstanceDocument<T = Record<string, any>> extends Stream {
 
   private _isReadOnly = false
 
-  get content(): T {
+  override get content(): T {
     return super.content
   }
 
   get metadata(): ModelInstanceDocumentMetadata {
     const metadata = this.state$.value.metadata
-    return { controller: metadata.controllers[0], model: metadata.model }
+    const controller = metadata.controllers[0]
+    if (!controller) throw new Error(`Controller is not present`)
```

Comment:
that shouldn't be possible, if controller wasn't present the handler should have failed when the stream was loaded